packages/hub/src/constants.ts

```diff
@@ -1,3 +1,5 @@
+require('../env');
```

Comment:
I noticed this was missing, was that intentional @kerzhner ?

Reply:
My personal opinion is that `require('../env')` is not a great way to set environment variables. But I am not sure I have a better solution in mind.

I tried to minimize the use of `require('../env')` to top-level entry points in the code base. `require('../env')` only needs to be called once during the execution of the program.

Reply:
I agree. However I noticed, for some reason, that `firebase-relay.ts` did not have the environment variables set when I ran it despite running from the entrypoint on `server.ts`